Makefile

```diff
@@ -41,7 +41,7 @@ else ifneq (,$(findstring meltingpot,$(example)))
 DOCKER_IMAGE_TAG=meltingpot
 endif
 
-IMAGE = $(DOCKER_IMAGE_NAME):$(DOCKER_IMAGE_TAG)
+IMAGE = $(DOCKER_IMAGE_NAME):$(DOCKER_IMAGE_TAG)-latest
```

Comment:
I was quite happy with not having the `latest` tag, but this is just a personal preference